src/components/Gouvernance/NoteDeContexte/AjouterNoteDeContext.tsx

```diff
@@ -0,0 +1,85 @@
+'use client'
+
+import { FormEvent, ReactElement, RefObject, useContext, useState } from 'react'
+
+import { clientContext } from '@/components/shared/ClientContext'
+import DrawerTitle from '@/components/shared/DrawerTitle/DrawerTitle'
+import EditeurDeTexte from '@/components/shared/EditeurDeTexteEnrichi/EditeurDeTexte'
+import { useRichTextEditor } from '@/components/shared/EditeurDeTexteEnrichi/hooks/useRichTextEditor'
+import { Notification } from '@/components/shared/Notification/Notification'
+import SubmitButton from '@/components/shared/SubmitButton/SubmitButton'
+
+export default function AjouterNoteDeContext({
```

Comment:
```suggestion
export default function AjouterNoteDeContexte({
```

Le fichier est à renommer également.